proposals/0024/GOVERNANCE.md

```diff
@@ -0,0 +1,30 @@
+# Governance
+
+This document describes how the Liberal contribution governance structure is implemented in the Tinkerbell community.
+
+## What is Liberal contribution?
+
+Under a liberal contribution model, the people who do the most work [in the prescribed way] are recognized as most influential, but this is based on current work and not historic contributions.
+Major project decisions are made based on a consensus seeking process (discuss major grievances) rather than pure vote, and strive to include as many community perspectives as possible.
+Popular examples of projects that use a liberal contribution model include Node.js and Rust <sup>[1]</sup>.
+
+## Community Roles
+
+This doc outlines the various roles in the Tinkerbell community.
+
+One of the overarching goals of having different roles in the community is to make sure to "allow good changes to enter the code with a minimum of fuss"<sup>[2]</sup>.
+The following criteria is to be used when considering and individual for a role.
+
+- a pattern of good judgement in regard to the code bases and to oneself
+- demonstrated history of "playing nice" in the community
+- amount and consistency of contributions and activity
+
+| Role        | Responsibilities | Requirements | Defined by |
+| -----       | ---------------- | ------------ | ------- |
+| Contributor | Submit contributions | signed DCO | n/a |
+| Member      | issue and discussion administration, PR approval | history of activity, good judgement, and contribution in the community and the repositories, Sponsored by 2 Committers | Tinkerbell GitHub org member |
```

Comment:
Would it be possible to call out a specific set of requirements that someone should meet before they ask for sponsorship for each step?

For example, as outlined by https://github.com/kubernetes/community/blob/master/community-membership.md - X PR's to become a committer.

Alternatively, would you rather let that discussion happen in a follow-up PR?

Reply:
```suggestion
| Member      | Issue and discussion administration, PR approval | history of activity, good judgement, and contribution in the community and the repositories, sponsored by 2 Committers | Tinkerbell GitHub org member |
```

Reply:
@tstromberg, If possible I would like a follow-up PR. 

I really do like the way the Kubernetes community has this laid out. I think we should definitely follow and make it explicit what requirements someone should meet before they ask for sponsorship for each step.